service/storage_service.cc

```diff
@@ -4659,7 +4659,10 @@ future<> storage_service::raft_rebuild(utils::optional_param sdc_param) {
         rtlogger.info("request rebuild for: {} source_dc={}", raft_server.id(), sdc_param);
         topology_mutation_builder builder(guard.write_timestamp());
         builder.set_session(session_id(guard.new_group0_state_id()));
-        auto source_dc = sdc_param.value_or("");
+        sstring source_dc = sdc_param.value_or("");
+        if (sdc_param.force() && !source_dc.empty()) {
+            source_dc += ":force";
```

Comment:
I hope we will not have DCs with name ending with ":force"

Reply:
agreed.  In initially had code that deals with that (by escaping the column if the dc name ended with `:force`) but I came to the conclusion that it's not worth the complication since the chances for that are really remote.